dev-python/plotly/plotly-4.10.0.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# blocker dev-python/retrying
+PYTHON_COMPAT=( python3_{6..7} )
+
+inherit distutils-r1 virtualx
+
+DESCRIPTION="Browser-based graphing library for Python"
+HOMEPAGE="https://plot.ly/python/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+
+LICENSE="MIT"
+SLOT="0"
+
+RDEPEND="${PYTHON_DEPS}
+	dev-python/retrying[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
+"
+DEPEND="${RDEPEND}"
```

Comment:
What for? Usually we don't need to bother with this, just at runtime, but maybe they are needed?

Reply:
Unsure how to do this without a lot of pain.
Don't see a harm in letting it stay.

Reply:
done